src/ui/elements.js

```diff
@@ -0,0 +1,86 @@
+//; -*- mode: rjsx;-*-
+/** @jsx jsx */
+import { jsx } from 'theme-ui';
+import { useTranslation } from 'react-i18next';
+
+
+// A styled `<input>`` element with a label. Displays errors and integrated with
+// `react-hook-form`.
+//
+// Props:
+// - `errors` and `register`: the two values returned by `useForm`
+// - `label`: the human readable string describing the field
+// - `name`, `type` and `rest`: passed as HTML attributes
+// - `required`: boolean, whether this field is required or may be empty
+// - `validate`: optional, a function validating the value and returning a
+//   string in the case of error.
+export const Input = ({
+  errors,
+  register,
+  label,
+  name,
+  required,
+  validate = undefined,
```

Comment:
```suggestion
  validate,
```

`undefined` is the default default parameter.